CMakeLists.txt

```diff
@@ -722,28 +724,32 @@ SET(targets)
 
 IF(PCRE2_BUILD_PCRE2_8)
   IF(BUILD_STATIC_LIBS)
-    ADD_LIBRARY(pcre2-8-static STATIC ${PCRE2_HEADERS} ${PCRE2_SOURCES} ${PROJECT_BINARY_DIR}/config.h)
+    ADD_LIBRARY(pcre2-8-static STATIC ${PCRE2_SOURCES})
```

Comment:
Comment 1:
I don't think this change belongs to this PR.
(And it may have downsides for developer which use an IDE and CMake to work on pcre2.

Reply:
Reverted.